java/jdbc/src/main/java/io/vitess/jdbc/ConnectionProperties.java

```diff
@@ -226,6 +231,11 @@
         "The password protecting the truststore file (if a password is set)",
         null,
         null);
+    private StringConnectionProperty trustStoreType = new StringConnectionProperty(
+        Constants.Property.TRUSTSTORE_TYPE,
+        "The type of store used for the trusted certificates (default is JKS)",
+        null,
+        null);
```

Comment:
Give default trustStoreType in connection property

StringConnectionProperty(String name, String description, String defaultValue, String[] allowableValuesToSet)